docs: OpenUnison integration; fix portal screen shot and linked to the open source website (#21179)
argoproj/argo-cd: docs/operator-manual/user-management/openunison.md

## Integrating OpenUnison and ArgoCD

> These instructions will take you through the steps of integrating OpenUnison and ArgoCD to support single sign-on and add a "badge" to your OpenUnison portal to create a single access point for both Kubernetes and ArgoCD.  These instructions assume you'll be using both ArgoCD's web interface and command line interface.  These instructions assume you are running [OpenUnison 1.0.20+](https://openunison.github.io/).

Reviewer: ```suggestion
These instructions will take you through the steps of integrating OpenUnison and ArgoCD to support single sign-on and add a "badge" to your OpenUnison portal to create a single access point for both Kubernetes and ArgoCD. These instructions assume you'll be using both Argo CD web interface and command line interface as well as running [OpenUnison 1.0.20+](https://openunison.github.io/).
```
